web/src/composebox_typeahead.ts

```diff
@@ -1179,6 +1191,56 @@ export function content_typeahead_selected(
     return beginning + rest;
 }
 
+function will_produce_broken_stream_topic_link(word: string): boolean {
+    return invalid_stream_topic_regex.test(word);
+}
+
+function get_stream_name(syntax: string): string {
```

Comment:
This function name is too generic; something like `get_stream_name_from_topic_link_syntax` would be more appropriate, though I've not checked exactly how it is used.

Reply:
This name matches what it does.
Changed.